db/seeds/db_populator.rb

```diff
@@ -34,7 +35,7 @@ def create_org(options_hash)
     casa_org = CasaOrg.find_or_create_by!(name: options.org_name) { |org|
       org.name = options.org_name
       org.display_name = options.org_name
-      org.casa_org_logo = logo
+      org.logo = logo
```

Comment:
For setting the logo here, we will need to [attach a file object](https://edgeguides.rubyonrails.org/active_storage_overview.html#attaching-file-io-objects):

```
org.logo.attach(io: File.open("media/src/images/logo.png"), filename: "pgcounty_logo.pdf")
```

Reply:
Disregard above comment. You can delete this part about setting the `logo`, since the code uses the default logo stored in the repo if no logo is attachec.